gstd/gstd_pipeline.c

```diff
@@ -159,6 +171,22 @@ gstd_pipeline_class_init (GstdPipelineClass * klass)
       "The event handler of the pipeline",
       GSTD_TYPE_EVENT_HANDLER, G_PARAM_READABLE | G_PARAM_STATIC_STRINGS);
 
+   properties[PROP_POSITION] =
+      g_param_spec_int64 ("position", "Position",
+      "The query position of the pipeline",
+      -1, /* Min value (-1 is used to indicate error, normal range 0 to G_MAXINT64 */
```

Comment:
Don't use magic numbers. There is a section at the top of the file with the defaults, minimums and maximums.

Reply:
The range was defined between 0 to G_MAXINT64